charts/s1-agent/templates/_helpers.tpl

```diff
@@ -187,6 +187,26 @@ Generate certificates for helper secret
 {{- dict "tls.crt" $tlsCert "tls.key" $tlsKey "ca.crt" $caCert | toYaml -}}
 {{- end -}}
 
+{{- define "helper_token.secret.create" -}}
+{{- empty .Values.secrets.helper_token | ternary "true" "" }}
```

Comment:
do we need the ternary? because `empty .Values.secrets.helper_token` will return bool

Reply:
yes, if helper token secret name was not set by a customer, we should create it.